app/controllers/catalog_controller.rb

```diff
@@ -432,7 +432,7 @@ def active_sort_fields
 
     config.add_sort_field("oldest_added") do |field|
       field.label = "oldest added"
-      field.sort = "date_created_dtsi asc"
+      field.sort = "date_published_dtsi asc, date_created_dtsi asc"
       field.if = ->(controller, field) { controller.current_user }
     end
```

Comment:
I suspect that admin doesn't really need "oldest added" at all -- they may instead need "created_at" both ascending and descending, now that "recently added" isn't really "created at" 

We could check with @apinkney0696 to be sure, but I think it's a good first bet, based on previous conversations.

Reply:
That's totally fine. I'd rather this be another PR though once we've checked with Annabel.

Reply:
OK, but what about adding created_at asc/desc (admin-only) in this PR then? I am 90% sure Annabel wants it.  That's what they used to get from "recently added" but won't anymore. 

How will we remember to bring this up with Annabel next week?

Reply:
Done!